FlowMagic-MockServer/MockServer/server.js

```diff
@@ -9,8 +9,10 @@ const { applicationScreenFlow } = require('./data.json')
 const fs = require('fs')
 const path = require('path')
 
-const dataFilePath = path.join(__dirname, 'data.json') // Adjust path to your data.json file
+const dataFilePath = path.join(__dirname, 'data.json')
+const testDataFilePath = path.join(__dirname, '/__tests__/testData.json')
 const data = JSON.parse(fs.readFileSync(dataFilePath, 'utf8'))
+const testData = JSON.parse(fs.readFileSync(testDataFilePath, 'utf8'))
```

Comment:
would it be easier to make dataFilePath and data load and store the test data in the test enviroment? 
```
if (process.env.NODE_ENV === 'test') {
    dataFilePath = path.join(__dirname, '/__tests__/testData.json')
    data = JSON.parse(fs.readFileSync(testDataFilePath, 'utf8'))
}
```
That way you don't have to have conditional statements to process testData separately.